### PR TITLE
Update CHANGELOG.md for 2.5.0

### PR DESCRIPTION
**Pull request by daschuer**
The next step is to polish it a bit and remove uninteresting entries.

### CODE REVIEW

**Comment by daschuer** [comment]
This is now up to date. Does one have a look? Next I will prepare a polish PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -114,6 +131,8 @@
   [#12020](https://github.com/mixxxdj/mixxx/pull/12020)
   [#13361](https://github.com/mixxxdj/mixxx/issues/13361)
   [#13364](https://github.com/mixxxdj/mixxx/pull/13364)
+  [#13958](https://github.com/mixxxdj/mixxx/pull/13958)
+  [#13958](https://github.com/mixxxdj/mixxx/issues/13100)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:135]:
```suggestion
  [#13100](https://github.com/mixxxdj/mixxx/issues/13100)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -404,13 +449,22 @@
...
+* Don't use deprecated "hotcue_1_enabled"/"hotcue_2_enabled" in unittests [#14026](https://github.com/mixxxdj/mixxx/pull/14026)
 * fix tsan issues
   [#13876](https://github.com/mixxxdj/mixxx/pull/13876)
   [#13869](https://github.com/mixxxdj/mixxx/issues/13869)
   [#13873](https://github.com/mixxxdj/mixxx/pull/13873)
   [#13875](https://github.com/mixxxdj/mixxx/pull/13875)
   [#13898](https://github.com/mixxxdj/mixxx/pull/13898)
-  [#13899](https://github.com/mixxxdj/mixxx/pull/[#13899](https://github.com/mixxxdj/mixxx/pull/13899))
+  [#13899](https://github.com/mixxxdj/mixxx/pull/13899)
+  [#13899](https://github.com/mixxxdj/mixxx/pull/13899)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:466]:
```suggestion
  [#13899](https://github.com/mixxxdj/mixxx/pull/13899)
```
Duplicate

**Review by JoergAtGithub — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Fixed the issues. I will merge this now and open a new one with the reorganization.

**Pull request merged**